util.js

```diff
@@ -105,9 +105,8 @@ exports.deprecate = function(fn, msg) {
 
 var debugs = {};
 var debugEnvRegex = /^$/;
-
-if (process.env.NODE_DEBUG) {
-  var debugEnv = process.env.NODE_DEBUG;
+if (process && process.env && process.env.NODE_DEBUG) {
```

Comment:
this will still error if `process` isn't defined.

and, it is *always* defined, since this is a _node_ module.